src/stores/connectors/eth1/sign.go

```diff
@@ -0,0 +1,294 @@
+package eth1
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"fmt"
+	"math/big"
+
+	"github.com/consensys/quorum-key-manager/pkg/errors"
+	"github.com/consensys/quorum-key-manager/pkg/ethereum"
+	"github.com/consensys/quorum-key-manager/src/stores/api/formatters"
+	quorumtypes "github.com/consensys/quorum/core/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/ethereum/go-ethereum/signer/core"
+	"golang.org/x/crypto/sha3"
+)
+
+var (
```

Comment:
move them in malleabilityECDSASignature() body since they're only used there?

Reply:
This code does not belong to my PR since it was only moved from `/src/stores/store/eth1` to here